geo/map_processing/map_structure.py

```diff
@@ -217,19 +220,9 @@ def cellid_from_polyline(self, polyline: Polygon, level: int) -> list:
         s2polygon = self.s2polygon_from_shapely_polyline(polyline)
         return self.get_s2cover_for_s2polygon(s2polygon, level)
 
-    def add_poi_to_graph(self, row: Series):
-        cells = row.cellids
-        poi = row.osmid
-        self.graph.add_poi(cells, poi)
-
-    def add_street_to_graph(self, row: Series):
-        cells = row.cellids
-        street = row.osmid
-        self.graph.add_street(cells, street)
-
     def __create_graph(self):
         '''
-        Helper funcion for creating graph
+        Helper funcion for creating graph.
         '''
 
         level = 18
```

Comment:
This should be an input to the function, not a constant set in the code.

Reply:
done